tests/test_date_parser.py

```diff
@@ -215,6 +219,44 @@ def test_dates_not_parsed(self, date_string):
         self.when_date_is_parsed()
         self.then_date_was_not_parsed()
 
+    @parameterized.expand([
```

Comment:
Tests should be explicit. We should have test with such configuration set and test for this behavior exactly. In general tests should only test one thing each.